packages/integration-tests/test/actions.spec.ts

```diff
@@ -44,18 +51,25 @@ const initWatcher = async () => {
 }
 
 const deploySimpleStorage = async () => {
-  const SimpleStorageJson = JSON.parse(fs.readFileSync('contracts/build/SimpleStorage.json').toString())
-  const SimpleStorageFactory = new ContractFactory(SimpleStorageJson.abi, SimpleStorageJson.bytecode, l2Wallet)
-  const dummy = await SimpleStorageFactory.deploy()
-  await dummy.deployTransaction.wait()
-  return SimpleStorageFactory.deploy()
+  const simpleStorageJson = JSON.parse(fs.readFileSync('contracts/build/SimpleStorage.json').toString())
+  const simpleStorageFactory = new ContractFactory(simpleStorageJson.abi, simpleStorageJson.bytecode, l2Wallet)
+  return simpleStorageFactory.deploy()
+}
+
+const deployERC20 = async (initialAmount, name, decimals, symbol) => {
+  const erc20Json = JSON.parse(fs.readFileSync('contracts/build/ERC20.json').toString())
```

Comment:
What do you think about breaking up this function `deployERC20` by having `erc20Json` and `ERC20Factory` be constants at the top of hte file and then just calling `ERC20Factory.deploy` in the `it` block

Reply:
👍 good call